azerothcore/azerothcore-wotlk
# fix(DB/creature_loot_template): Quest item drop chance - 'Guardian of the Monument'

## CHANGES PROPOSED:
-  Fix drop chance of the quest item


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3788



## TESTS PERFORMED:
- None ( I'm sorry, but unfortunately I can't run the wow client at the moment)




## HOW TO TEST THE CHANGES:

- Take the quest: https://www.wowhead.com/quest=11059/guardian-of-the-monument
- Make sure the quest item [Apexis Guardian's Head] is always dropped



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
